document-generator-api/pom.xml

```diff
@@ -21,11 +21,11 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         
         <!-- JDK Version -->
-        <jdk.version>1.8</jdk.version>
+        <jdk.version>21</jdk.version>
 
-        <apache-http-components.version>4.5.13</apache-http-components.version>
+        <apache-http-components.version>5.4-beta1</apache-http-components.version>
```

Comment:
Please double check if we are ok to have beta versions. I'd say go for the latest stable version.

Reply:
Changed to 5.4